README.md

```diff
@@ -55,100 +59,196 @@ base16-black-metal-mayhem
 base16-gruvbox-dark-medium
 base16-gruvbox-dark-pale
 base16-gruvbox-dark-soft
 base16-gruvbox-light-hard
 base16-gruvbox-light-medium
 base16-gruvbox-light-soft
+base16-gruvbox-material-dark-hard
+base16-gruvbox-material-dark-medium
+base16-gruvbox-material-dark-soft
+base16-gruvbox-material-light-hard
+base16-gruvbox-material-light-medium
+base16-gruvbox-material-light-soft
+base16-hardcore
 base16-harmonic-dark
 base16-harmonic-light
 base16-heetch-light
 base16-heetch
 base16-helios
 base16-hopscotch
 base16-horizon-dark
+base16-horizon-light
+base16-horizon-terminal-dark
+base16-horizon-terminal-light
+base16-humanoid-dark
+base16-humanoid-light
 base16-ia-dark
 base16-ia-light
 base16-icy
 base16-irblack
 base16-isotope
+base16-kanagawa
+base16-katy
+base16-kimber
+base16-lime
 base16-macintosh
 base16-marrakesh
-base16-materia
 base16-material-darker
 base16-material-lighter
 base16-material-palenight
 base16-material-vivid
 base16-material
+base16-materia
 base16-mellow-purple
 base16-mexico-light
 base16-mocha
 base16-monokai
+base16-nebula
 base16-nord
-base16-ocean
+base16-nova
 base16-oceanicnext
-base16-one-light
+base16-ocean
 base16-onedark
+base16-one-light
 base16-outrun-dark
+base16-pandora
 base16-papercolor-dark
 base16-papercolor-light
 base16-paraiso
+base16-pasque
 base16-phd
 base16-pico
+base16-pinky
 base16-pop
 base16-porple
+base16-primer-dark-dimmed
+base16-primer-dark
+base16-primer-light
+base16-purpledream
+base16-qualia
 base16-railscasts
 base16-rebecca
-base16-schemer-dark
-base16-schemer-medium
+base16-rose-pine-dawn
+base16-rose-pine-moon
+base16-rose-pine
+base16-sagelight
+base16-sakura
+base16-sandcastle
 base16-seti
+base16-shadesmear-dark
+base16-shadesmear-light
+base16-shades-of-purple
 base16-shapeshifter
+base16-silk-dark
+base16-silk-light
 base16-snazzy
+base16-solarflare-light
 base16-solarflare
 base16-solarized-dark
 base16-solarized-light
+base16-spaceduck
 base16-spacemacs
+base16-stella
+base16-still-alive
+base16-summercamp
 base16-summerfruit-dark
 base16-summerfruit-light
 base16-synth-midnight-dark
+base16-synth-midnight-light
+base16-tango
+base16-tender
+base16-tokyo-city-dark
+base16-tokyo-city-light
+base16-tokyo-city-terminal-dark
+base16-tokyo-city-terminal-light
+base16-tokyodark-terminal
+base16-tokyodark
+base16-tokyo-night-dark
+base16-tokyo-night-light
+base16-tokyo-night-storm
+base16-tokyo-night-terminal-dark
+base16-tokyo-night-terminal-light
+base16-tokyo-night-terminal-storm
 base16-tomorrow-night-eighties
 base16-tomorrow-night
 base16-tomorrow
 base16-tube
 base16-twilight
 base16-unikitty-dark
 base16-unikitty-light
+base16-unikitty-reversible
+base16-uwunicorn
+base16-vice
+base16-vulcan
+base16-windows-10-light
+base16-windows-10
+base16-windows-95-light
+base16-windows-95
+base16-windows-highcontrast-light
+base16-windows-highcontrast
+base16-windows-nt-light
+base16-windows-nt
 base16-woodland
 base16-xcode-dusk
-base16-zenburn
-```
+base16-zenburn```
+````
```

Comment:
The formatted for the backticks is a bit off, only use 3 at the top and bottom of the codeblock.

Reply:
Fixed, thanks for noticing the typo!